installer/manifests/keptn/templates/core.yaml

```diff
@@ -52,7 +52,7 @@ metadata:
     app.kubernetes.io/name: bridge
 type: Opaque
 data:
-  BASIC_AUTH_USERNAME: {{ "keptn" | b64enc | quote }}
+  BASIC_AUTH_USERNAME: {{ default "keptn" .Values.bridge.keptnUsername | b64enc | quote }}
```

Comment:
I think we should also add a block above that fetches. the username from the secret if it already exists, like it's done for the bridge password. So something like this:
```
{{- $bridgePassword := (randAlphaNum 20) | b64enc | quote }}
{{- $bridgeUsername := default "keptn" .Values.bridge.keptnUsername | b64enc | quote }}
{{- $bridgesecret := (lookup "v1" "Secret" .Release.Namespace "bridge-credentials") }}
{{- if $bridgesecret }}
  {{- $bridgeUsername = index $bridgesecret.data "BASIC_AUTH_USERNAME" }}
  {{- $bridgePassword = index $bridgesecret.data "BASIC_AUTH_PASSWORD" }}
{{- end -}}
```

And maybe with that it also makes sense to rename the value to `basicAuthUsername` to make a clear distinction between OAuth and HTTP basic auth